x-pack/legacy/plugins/index_management/__jest__/client_integration/helpers/setup_environment.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import axios from 'axios';
+import axiosXhrAdapter from 'axios/lib/adapters/xhr';
+import { init as initHttpRequests } from './http_requests';
+import { AppContextProvider } from '../../../public/application/app_context';
+import { httpService } from '../../../public/application/services/http';
+import { breadcrumbService } from '../../../public/application/services/breadcrumbs';
+import { documentationService } from '../../../public/application/services/documentation';
+import { notificationService } from '../../../public/application/services/notification';
+import { ExtensionsService } from '../../../public/services';
+import { UiMetricService } from '../../../public/application/services/ui_metric';
+import { setUiMetricService } from '../../../public/application/services/api';
+import { setExtensionsService } from '../../../public/application/store/selectors';
+
+/* eslint-disable @kbn/eslint/no-restricted-paths */
+import { notificationServiceMock } from '../../../../../../../src/core/public/notifications/notifications_service.mock';
+import { docLinksServiceMock } from '../../../../../../../src/core/public/doc_links/doc_links_service.mock';
+
+const mockHttpClient = axios.create({ adapter: axiosXhrAdapter });
+
+export const services = {
+  extensionsService: new ExtensionsService(),
+  uiMetricService: new UiMetricService('index_management'),
+};
+services.uiMetricService.setup({ reportUiStats() {} } as any);
+setExtensionsService(services.extensionsService);
+setUiMetricService(services.uiMetricService);
+const appDependencies = { services, core: {}, plugins: {} } as any;
+
+export const setupEnvironment = () => {
+  // Mock initialization of services
+  // @ts-ignore
+  httpService.setup(mockHttpClient);
+  breadcrumbService.setup(() => undefined);
+  documentationService.setup(docLinksServiceMock.createStartContract());
+  notificationService.setup(notificationServiceMock.createStartContract());
```

Comment:
should this be `notificationServiceMock.createSetupContract()`?

Reply:
They are both the same, but for consistency let's use the `createSetupContract ()` 👍